src/qml/General/AboutPage.qml

```diff
@@ -16,9 +16,169 @@
 
 import QtQuick 2.9
 import QtQuick.Controls 2.2
+// Units & font sizes
+import LunaNext.Common 0.1
 
 import "../Common"
 
 BasePage {
 
+    property alias deviceName:        deviceNameLabel.value
+    property alias deviceSerial:      deviceSerialLabel.value
+    property alias deviceWifiAddress: deviceWifiAddrLabel.value
```

Comment:
Same here not aligned with items below

Reply:
I wanted to do 2 groups here; I find that having too many things aligned is actually harder to read sometimes.

Reply:
OK